crates/bevy_ecs/src/schedule/stage.rs

```diff
@@ -1607,392 +1594,6 @@ mod tests {
         stage.run(&mut world);
     }
 
-    #[test]
-    fn ambiguity_detection() {
```

Comment:
This test was removed, rather than fixed, because it was generally a disaster to debug / refactor / use. More tests will be added in follow-up PRs, promise.

Reply:
that makes me quite sad